### DISCUSSION TITLE
Set the solver precision for the linear solver model,

### DISCUSSION THREAD

**Discussion by omidziaee** [optimatika/ojAlgo#528] [General, ojAlgo and related code]
Is there anyway to set the solver precision for the linear optimization problem with OjAlgo? I see the result has a lot of decimal points. Is there a way to set it like to three decimal points?

**Comment by apete** [comment]
Try setting `model.options.solution` to something suitable.

**Comment by omidziaee** [comment]
Does it have effect on the solution time? Like less precision lower solution time?
